.gitignore

```diff
@@ -34,3 +34,6 @@ src/**/Manifest.toml
 
 # Julia System Images
 *.so
+
+# Slurm Scripts
+*.sh
```

Comment:
```suggestion
```
These are not generated by the Machine, by Julia, or by the OS, and so I don't think they should be ignored by git.